ios/Podfile

```diff
@@ -1,9 +1,13 @@
 require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
-platform :ios, '10.0'
+
+source 'https://github.com/react-native-tvos/react-native-tvos-podspecs.git'
```

Comment:
Is it safe to track `master` or should we pin to a known version?

Reply:
Good question, but looking at the repo it's clear how static the podspec is. There is only a master branch for that repo. Fairly low risk, although we could fork it to be extra safe, perhaps worth noting in an issue for prior to release?

Reply:
Let's just leave a comment here explaining why we're happy to track master.

Reply:
Added!